### ISSUE TITLE
Model Task

### ISSUE THREAD

**Issue by Ademola111** [BuildForSDG/good-health-server#2]
Hi, be informed that you are to carry out the user model task. kindly go through the blow the feature below.

User model:
id :int primary key
first_name :string required 
last_name :string required
email :string required
password :string required
phone_number :string 
avatar:image

**Comment by elizabethadegbaju** [comment]
There is an inbuilt django user model so I think we don't need to create one. we can extend it using OneToOneField and add the image and phone number though.

**Comment by Ademola111** [comment]
Okay since we have that in place then I'll be assigning a new task for Idowu or whats your suggestion

**Comment by tonymj76** [comment]
yeah i know about the user model but it uses username for login verification which is not cool at all.. on the bright side you can extend it and allow but email or username  can be use

**Comment by elizabethadegbaju** [comment]
oh you didn't express any opposition to that previously. if we wanted to customize our user model then we should have done it first before any migrations. we can still clear our previous migrations and delete our existing database before customizing the user though. Username for logging in isn't that bad but we can change it to email if that's preferred.
